src/analyzer/analyzerwaveform.h

```diff
@@ -131,13 +132,15 @@ struct WaveformStride {
     }
 
     int m_position;
+    int m_stemCount;
     double m_length;
     double m_averageLength;
     int m_averagePosition;
     int m_averageDivisor;
 
     float m_overallData[ChannelCount];
     float m_filteredData[ChannelCount][FilterCount];
+    float m_stemData[ChannelCount][4];
```

Comment:
`ChannelCount` and `4` are not self explaining here. Can you add a constexpr kStemCount = 4 in waveform.h

Reply:
Please move the constexpr to waveform.h, where it can also be used: https://github.com/mixxxdj/mixxx/blob/a0ab0c3216e0c1353ff87aca2ac27776fd9ca018/src/waveform/waveform.h#L24